gqueries/general/primary_demand/total_primary_demand/total_primary_demand_of_hydro_electricity.gql

```diff
@@ -0,0 +1,11 @@
+- query =
+    SUM(
+      V(
+        FILTER(
+          G(primary_energy_demand),
+          "merit_order && (merit_order.group.to_s.gsub(/\s/, '') == 'river')"
```

Comment:
First, in terms of syntax, the following is more legible and also works:
```
"merit_order && (merit_order.group == :river)"
```

Second, through this query we're missing `energy_power_hydro_mountain`. Shouldn't we rewrite it to be in line with wind and solar?

```
V(FILTER(G(primary_energy_demand),"water_input_conversion > 0.0"),primary_demand_of_electricity)
```